lib/node_modules/@stdlib/math/base/special/boxcox/lib/main.js

```diff
@@ -0,0 +1,53 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var abs = require( '@stdlib/math/base/special/abs' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var ln = require( '@stdlib/math/base/special/ln' );
+var expm1 = require( '@stdlib/math/base/special/expm1' );
+var EPS = require( '@stdlib/constants/math/float64-eps' );
+
+
+// MAIN //
+
+/**
+ * Computes the Box-Cox transformation.
+ *
+ * @param {number} x - input value
+ * @param {number} lambda - input value
+ * @returns {number} Box-Cox transformation
+ */
+function boxcox(x, lambda) {
+	if ( isnan( x ) || isnan( lambda ) ) {
+		return NaN;
+	}
+
+	if ( abs( lambda ) < EPS ) {
```

Comment:
Any insight into why SciPy choses `1.0e-19` here rather than `EPS`?

Reply:
I have wondered about that as well. The documentation talks about `eps` and the `boxcox1p` uses some seemingly arbitrary values for bounds checking as well. I do not know the reasoning at current time though.

Reply:
We can probably give their implementation the benefit of the doubt. I don't believe their usage of `eps` is meant to be floating-point epsilon, but rather just a threshold. In which case, we can just follow their lead on this and use `1.0e-19`.